File: cadwiki-nuget/UnitTests/cadwiki.DllReloader/TestAcadAssemblyUtils.vb
Imports System.Reflection
Imports cadwiki.DllReloader.AutoCAD

<TestClass()> Public Class TestAcadAssemblyUtils

    <TestMethod()> Public Sub Test_GetAppAssemblySafeley_WithNothing_ShouldReturnNothing()
        Dim expected As Object = Nothing
        Dim actual As Assembly = AcadAssemblyUtils.GetAppObjectSafely(Nothing)
        Assert.AreEqual(expected, actual)
    End Sub

End Class

File: cadwiki-nuget/UnitTests/cadwiki.DllReloader/TestAcadAssemblyUtils.cs
using System.Reflection;
using cadwiki.DllReloader.AutoCAD;
using Microsoft.VisualStudio.TestTools.UnitTesting;

namespace UnitTests
{

    [TestClass()]
    public class TestAcadAssemblyUtils
    {

        [TestMethod()]
        public void Test_GetAppAssemblySafeley_WithNothing_ShouldReturnNothing()
        {
            object expected = null;
            Assembly actual = (Assembly)AcadAssemblyUtils.GetAppObjectSafely(null);
            Assert.AreEqual(expected, actual);
        }

    }
}